charts/datadog/values.yaml

```diff
@@ -537,6 +537,8 @@ datadog:
 
       # datadog.apm.instrumentation.targets -- Enable target based workload selection (preview).
       # Requires Cluster Agent 7.64.0+
+      #
+      # ddTraceConfigs[..]valueFrom requires agent 7.66.0+.
```

Comment:
Is there any way we can enforce this through validation? 🤔

Reply:
Given that you can use _any_ tag, I'm not sure that we can :(